packages/twenty-website/src/content/developers/local-setup.mdx

```diff
@@ -138,19 +140,35 @@ You can access the database at [localhost:5432](localhost:5432), with user `twen
   </ArticleTab>
 </ArticleTabs>
 
+## Step 4: Set up a Redis Database (cache)
```

Comment:
Since we offer a docker PostgreSQL. Can we offer a docker option also? @charlesBochet

Reply:
@hozza We actually want to get rid of docker in the contributor guide.
While I personally think docker is the best way, most of the contributors are not at ease with it and this was generating too much friction on the install (many thread on Discord). We've chosen not to maintain docker containers ourselves.

Setting up redis with docker is straightforward and I believe that contributors wanting to provision it through docker will manage to do it by themselves.

I would say docker setup should not be part of the local setup guide and we will (or the community!) will likely write a blog post about it at some point

Reply:
Thanks, your comments here may help to explain this as well https://github.com/twentyhq/twenty/issues/7370